packages/poi/test/config.test.js

```diff
@@ -197,4 +201,21 @@ describe('get webpack config', () => {
       expect(config.entry.foo).toEqual(['foo', 'bar'])
     })
   })
+
+  describe('build component', () => {
+    it('respects component option', async () => {
```

Comment:
I think we can just require `html-webpack-plugin` here?

Reply:
that would probably be clearer, yes